Userland/Libraries/LibWebView/InspectorClient.cpp

```diff
@@ -44,9 +44,10 @@ static String style_sheet_identifier_to_json(Web::CSS::StyleSheetIdentifier cons
         identifier.url.value_or("undefined"_string)));
 }
 
-InspectorClient::InspectorClient(ViewImplementation& content_web_view, ViewImplementation& inspector_web_view)
+InspectorClient::InspectorClient(ViewImplementation& content_web_view, ViewImplementation& inspector_web_view, bool& is_windowed)
```

Comment:
Why are we passing the boolean by reference?

Reply:
honestly not too sure still learning c++ and getting a grip on pointers vs refs and such, sorry!